### PR TITLE
added pcsx-rearmed core

### PR DESCRIPTION
**Pull request by eibma**

### CODE REVIEW

File: project/cmake/addons/addons/game.libretro.pcsx-rearmed/platforms.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+linux windows darwin
```
Comment (garbear) [inline project/cmake/addons/addons/game.libretro.pcsx-rearmed/platforms.txt]: have you tested on all 3? I can test when I'm done with input

**Comment by eibma** [comment]
no just on linux - would you like to have a pull request with only linux enabled? I could also set up a windows environment but darwin (which is OSX i guess?) is not possible for me

**Comment by sigboe** [comment]
I have a Macbook pro, if someone would send me a binary to test. I don't
have a build environment set up, but I suppose I can do that as well, maybe
not today but soon.

**Comment by garbear** [comment]
@sigboe thanks for the offer. i develop on all three in parallel, each has their strengths and weaknesses, so i can generate binaries for all 3

@notspiff where can we upload zips for an add-on repo? technically we can't distribute most cores with Kodi, and there's already logic in place for just-in-time installation (though I think it broke in the rebase, see [this TODO](https://github.com/garbear/xbmc/commit/42424c4))

the addon.xml files for game add-on need some minor changes. compare [game.libretro.pcsx-rearmed/addon.xml](https://github.com/kodi-game/game.libretro.pcsx-rearmed/blob/master/game.libretro.pcsx-rearmed/addon.xml) to [game.libretro.nestopia/addon.xml](https://github.com/kodi-game/game.libretro.nestopia/blob/master/game.libretro.nestopia/addon.xml). &lt; allowvfs> became &lt;supports_vfs> and &lt;blockextract> was dropped. Also, I renamed nestopia to "NES (Nestopia)", the idea being to highlight the platform, but I'm not sure I like this change because the emulator selection dialog for NES roms only shows NES emulators anyways

**Comment by garbear** [comment]
input is complete. i've updated the add-on fluff, see https://github.com/kodi-game/game.libretro.pcsx-rearmed/compare/1b7792b...master. when I get a chance I'll test out pcsx-rearmed on windows and osx.

**Comment by garbear** [comment]
Cherry picked into master. thanks!

**Pull request closed without merging**